js-weather/js/script.js

```diff
@@ -0,0 +1,67 @@
+let xhr = new XMLHttpRequest();
+let AppId = '4b5774e9f3d2a07b84f0f2f88e486224';
+
+document.addEventListener('DOMContentLoaded', function () {
```

Comment:
```suggestion
document.addEventListener('DOMContentLoaded', () => {
```
functionキーワードは全廃してアロー関数にしましょう！

参考

[アロー関数のみにしていく](https://qiita.com/shibukawa/items/19ab5c381bbb2e09d0d9#%E9%96%A2%E6%95%B0%E5%AE%A3%E8%A8%80)